lib/improver/cli/orographic_enhancement.py

```diff
@@ -161,18 +97,31 @@ def process(temperature, humidity, pressure, wind_speed, wind_dir, orography):
         orography (iris.cube.Cube):
             Cube containing height of orography above sea level on high
             resolution (1 km) UKPP domain grid.
+        boundary_height (float):
```

Comment:
If you have set it as 1000.0 above, you should probably put here that the default is 1000.0?

Reply:
Current advice is not to have `Default is ...` in the clized docstrings.  The function definition gets rendered in the documentation and contains the default value, so this would just be repeating information.

Reply:
I did not realise that. Previously we have had the default in, so thought it might be useful. However it seems not!